fluss-server/src/main/java/com/alibaba/fluss/server/kv/KvTablet.java

```diff
@@ -259,6 +263,36 @@ public void registerMetrics(BucketMetricGroup bucketMetricGroup) {
         metricGroup.meter(
                 MetricNames.KV_PRE_WRITE_BUFFER_TRUNCATE_AS_ERROR_RATE,
                 new MeterView(kvPreWriteBuffer.getTruncateAsErrorCount()));
+
+        // Initialize RocksDB metrics reporter
+        inWriteLock(
+                kvLock,
```

Comment:
I think we don't need to initialize `rocksDBMetricsCollector` in lock, because there is no concurrency problem here. Besides, I think it's better to move the `rocksDBMetricsCollector` initialization to the constructor to simplify the logic and also make the `rocksDBMetricsCollector` to be final.